.github/ISSUE_TEMPLATE.md

```diff
@@ -4,7 +4,7 @@
 
 We want to keep signal strong in the GitHub issue tracker – to make sure that it remains the best place to track issues that affect the development of Relay.
 
-If you have a question on how to use Relay, please [post it to Stack Overflow](https://stackoverflow.com/questions/ask?tags=relayjs) with the tag [#relayjs](http://stackoverflow.com/questions/tagged/relayjs).
+If you have a question on how to use Relay, please get in touch with community members through one of the channels listed [here](https://facebook.github.io/relay/en/help.html) instead.
```

Comment:
I was tempted to add ‘Tickets for such usage questions _will_ be closed.’ to make it more explicit, but wanted to know what y’all prefer?